website/versioned_docs/version-0.67/release-testing.md

```diff
@@ -0,0 +1,122 @@
+---
+id: release-testing
+title: Release Testing
+---
+
+## Pre-requisites
+
+- Have clone of `react-native` repo and be on the release candidate branch.
+
+  ```bash
+  # Checkout relevant branch
+  react-native$ git checkout 0.66-stable
+  react-native$ yarn install
+
+  # I needed a .watchmanconfig because `npm start` would fail without it.
+  react-native$ echo '{}' > .watchmanconfig
+  ```
+
+- Have Android and iOS development environment set-up. Follow instuctions for macOS/iOS and macOS/Android from [React Native docs](https://reactnative.dev/docs/environment-setup)
```

Comment:
```suggestion
- Have Android and iOS development environment set-up. Follow instructions for macOS/iOS and macOS/Android from the [Environment Setup](/docs/environment-setup) guide.
```

We should use the internal links to avoid re-render on navigation.